src/structures/Emoji.js

```diff
@@ -190,7 +190,7 @@ class Emoji extends Base {
   }
 
   /**
-   * When concatenated with a string, this automatically returns the emoji mention rather than the object.
+   * When concatenated with a string, this automatically concatenates the emoji's mention instead of the Emoji object.
```

Comment:
returns is the correct terminology, the other ones should be changed to reflect that

Reply:
I thought about that and just went with whatever the majority was using. But yeah, `returns` makes more sense.